src/main/java/org/jabref/gui/fieldeditors/LinkedEntriesEditor.java

```diff
@@ -71,6 +71,8 @@ public LinkedEntriesEditor(Field field, BibDatabaseContext databaseContext, Sugg
         entryLinkField.setNewItemProducer(searchText -> viewModel.getStringConverter().fromString(searchText));
         entryLinkField.setMatcher((entryLink, searchText) -> entryLink.getKey().toLowerCase().startsWith(searchText.toLowerCase()));
         entryLinkField.setComparator(Comparator.comparing(ParsedEntryLink::getKey));
+
+        entryLinkField.setPlaceholder(null);
```

Comment:
Do you really need `null` here? Shouldn't `""` be used?

Reply:
> Do you really need `null` here? Shouldn't `""` be used?

No, the `setPlaceholder `method takes a `Node`, not a `String` I use this method to hide the dropdown list when the search result is empty.